ui/components/ui/tabs/tabs.component.js

```diff
@@ -41,6 +44,10 @@ const Tabs = ({
       setActiveTabIndex(tabIndex);
       onTabClick?.(tabKey);
     }
+
+    if (tabKey === 'nfts') {
+      dispatch(detectNfts());
```

Comment:
Oh cool.  Does this trigger for both the nft tab on the main page _and_ the asset picker tab because they share this key?